[2025-01-10T17:25:04.674+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-10T17:25:04.689+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:24:00+00:00 [queued]>
[2025-01-10T17:25:04.698+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:24:00+00:00 [queued]>
[2025-01-10T17:25:04.699+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-10T17:25:04.714+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): append_to_database> on 2025-01-10 17:24:00+00:00
[2025-01-10T17:25:04.719+0000] {standard_task_runner.py:72} INFO - Started process 1541 to run task
[2025-01-10T17:25:04.722+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'stock_information', 'append_to_database', 'scheduled__2025-01-10T17:24:00+00:00', '--job-id', '163', '--raw', '--subdir', 'DAGS_FOLDER/vnstock_stream.py', '--cfg-path', '/tmp/tmpqrl2yj82']
[2025-01-10T17:25:04.725+0000] {standard_task_runner.py:105} INFO - Job 163: Subtask append_to_database
[2025-01-10T17:25:04.764+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:24:00+00:00 [running]> on host 453ac4defedc
[2025-01-10T17:25:04.849+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TuanDz' AIRFLOW_CTX_DAG_ID='stock_information' AIRFLOW_CTX_TASK_ID='append_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T17:24:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-10T17:24:00+00:00'
[2025-01-10T17:25:04.851+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-10T17:25:20.666+0000] {logging_mixin.py:190} INFO - Lỗi kết nối db: ('HYT00', '[HYT00] [Microsoft][ODBC Driver 18 for SQL Server]Login timeout expired (0) (SQLDriverConnect)')
[2025-01-10T17:25:20.668+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-10T17:25:20.675+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-10T17:25:20.677+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_information, task_id=append_to_database, run_id=scheduled__2025-01-10T17:24:00+00:00, execution_date=20250110T172400, start_date=20250110T172504, end_date=20250110T172520
[2025-01-10T17:25:20.721+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-10T17:25:20.739+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-10T17:25:20.744+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
